src/annotator/sidebar.js

```diff
@@ -63,54 +63,50 @@ export default class Sidebar extends Guest {
   constructor(element, config) {
     super(element, config);
 
-    let externalFrame;
-    let frame;
-    let hypothesisSidebar; // refers to <hypothesis-sidebar> element
-
-    if (config.externalContainerSelector) {
-      externalFrame =
-        document.querySelector(config.externalContainerSelector) || element;
+    this.iframe = createSidebarIframe(config);
+
+    const {
+      externalContainerSelector,
+      theme,
+      openSidebar,
+      annotations,
+      query,
+      group,
+    } = config;
```

Comment:
My subjective view here is that this use of destructuring isn't a win for readability, because some of these variables are used only much later in the function and it is now less obvious that they are just values from the config.

Reply:
I agree, I will leave them as they were.